.github/workflows/ingestion_client.yaml

```diff
@@ -31,8 +38,55 @@ jobs:
             echo "Generated ARM template does not match the existing main.json. Please update main.json accordingly." >&2
             exit 1
           fi
-  build:
+
+  test:
+    runs-on: ubuntu-latest
     needs: convert_bicep
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '8.0.x'
+
+      - name: Install coverlet
+        run: dotnet tool install --global coverlet.console
+
+      - name: Install reportgenerator
+        run: dotnet tool install --global dotnet-reportgenerator-globaltool
+
+      - name: Run Unit Tests and Collect Coverage
+        run: |
+          dotnet test ./samples/ingestion/ingestion-client/Tests --configuration Release /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura
```

Comment:
please add --filter "TestCategory=UnitTest" to dotnet test command (see https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=mstest#syntax)

Reply:
Added